build.xml

```diff
@@ -157,7 +157,7 @@
 
         <testng suitename="htsjdk-single-test" classpathref="classpath" outputdir="${test.output}"
                 verbose="${testng.verbosity}">
-            <jvmarg line="-Xmx512M ${debug.jvm.args}"/>
+            <jvmarg line="-Xmx2g ${debug.jvm.args}"/>
```

Comment:
Thanks for catching this!  I had changed it when I was experimenting with using a pair of BitSets that sucked up quite a bit of memory and forgot to change it back.